components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -773,6 +775,8 @@
         var groupSelection = this._remoteGroupLabel.replace("__serverID__", this._selectedServer);
         $$('#group-selection h2')[0].innerHTML = groupSelection;
 
+        if (serverData.registrationURL != "") {$('user-selection').insert(new Element('p', {'class' : 'intro xHint'}).insert("$!escapetool.javascript($services.localization.render('phenotips.PushPatient.remoteRegistration'))".replace("__serverID__", this._selectedServer).replace("__registrationUrl__", "&lt;a target=\"_blank\" href=\""+serverData.registrationURL+"\"&gt;"+serverData.registrationURL+"&lt;/a&gt;")));}
```

Comment:
Additionally, the line about signing up should disappear once the user is logged in.